src/docs/testing/unit-testing.md

```diff
@@ -29,14 +29,34 @@ import { MyCmp } from '../my-cmp';
 it('should render my component', async () => {
   const page = await newSpecPage({
     components: [MyCmp],
-    html: `<my-cmp></my-cmp>`,
+    html: `<my-cmp title="Hello World"></my-cmp>`,
   });
   expect(page.root).toEqualHtml(`
-    <my-cmp>Success!</my-cmp>
+    <my-cmp>Hello World</my-cmp>
```

Comment:
If we change this assertion, we should also update the [text above it](https://github.com/ionic-team/stencil-site/pull/729/files#diff-bc68864585479eef3eceee2c25bf4efcd68df4981723ee307e7e06f715abe7b0R22):

> In this example, when the component `MyCmp` renders it sets its text content as "Success!". The matcher `toEqualHtml()` is then used to ensure the component renders as expected.

Alternatively, we could also roll the changes back on lines 32 & 35